chart/templates/deployment-sidekiq.yaml

```diff
@@ -6,8 +6,12 @@ metadata:
     {{- include "mastodon.labels" . | nindent 4 }}
 spec:
   {{- if not .Values.autoscaling.enabled }}
+  {{- if .Values.mastodon.sidekiq.replicaCount }}
```

Comment:
Should we use `| default` instead of conditionals here? or is there some reason not to?

Reply:
Good point, let me patch that real quick :)